node_helper.js

```diff
@@ -158,6 +169,26 @@ module.exports = NodeHelper.create({
         return dates.map(({date, games}) => games.map(game => ({...game, gameDay: date}))).flat();
     },
 
+    /**
+     * @function fetchPlayoffs
+     * @description Retrieves playoff data from the API.
+     * @async
+     *
+     * @returns {object} Raw playoff data from API endpoint.
+     */
+    async fetchPlayoffs() {
+        const response = await fetch(BASE_PLAYOFF_URL);
+
+        if (!response.ok) {
+            console.error(`Fetching NHL playoffs failed: ${response.status} ${response.statusText}. Url: ${BASE_PLAYOFF_URL}`);
```

Comment:
```suggestion
            console.error(`Fetching NHL playoffs failed: ${response.status} ${response.statusText}`);
```
The URL is static and doesn't add additional information here as it is clear from the logs that the playoff request failed.